coalib/coala_json.py

```diff
@@ -28,12 +31,38 @@ def main():
     args = arg_parser.parse_args()
 
     log_printer = None if args.text_logs else ListLogPrinter()
-    results, exitcode, _ = run_coala(log_printer=log_printer, autoapply=False)
+    JSONEncoder = create_json_encoder(use_relpath=args.relpath)
+
+    if args.show_bears:
+        try:
+            local_bears, global_bears = get_filtered_bears(
+                args.filter_by_language, log_printer)
 
-    retval = {"results": results}
+            bears = inverse_dicts(local_bears, global_bears)
+            results = []
+            for bear, sections in sorted(bears.items(),
+                                         key=lambda bear_tuple:
+                                         bear_tuple[0].name):
+                metadata = bear.get_metadata()
+                results.append({"Name": bear.name,
+                                "Description": metadata.desc,
+                                "Supported languages": bear.LANGUAGES,
```

Comment:
@sils1297 I think that appending `bear` isn't a good idea. The JSONEncoder will recursively go through all attributes in a bear, but some attributes like [this](https://github.com/coala-analyzer/coala-bears/blob/master/bears/c_languages/ClangComplexityBear.py#L23) will break the JSONEncoder. We can handle such cases in JSONEncoder but then we'll have to do this every time something isn't serializable.
I think we should stick to only outputting the stuff we want (like in `coala`). What do you say?